app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,70 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  before_action :signed_in_only!, only: %i[new create]
+
+  # GET /posts
+  def index
+    @posts = Post.all.order('created_at DESC')
+    @post = Post.new
+  end
+
+  # GET /posts/1
+  def show; end
+
+  # GET /posts/new
+  def new
+    @post = current_member.posts.build
+  end
```

Comment:
- [ ] as the previous TSE mentioned, please change this `@post` to `@post = Post.new`

I've tested it, and the website still functions with no issue after this change :)

Reply:
SOLVED